This test fails:

```

======================================================================
FAIL: test_smart_matrix (test_SmartMatrix.TestSmartMatrix)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/missionpinball/mpf/mpf/tests/test_SmartMatrix.py", line 70, in test_smart_matrix
    call(b'\x01\x00\x01\x02\x03')                               # frame
  File "/usr/lib/python3.5/unittest/mock.py", line 824, in assert_has_calls
    ) from cause
AssertionError: Calls not found.
Expected: [call(b'\xba\x11\x00\x03\x14\x7f\x00\x00'), call(b'\x01\x00\x01\x02\x03')]
Actual: [call(bytearray(b'\x01\x00\x01\x02\x03'))]

```

--- a/mpf/tests/test_SmartMatrix.py
+++ b/mpf/tests/test_SmartMatrix.py
@@ -60,13 +60,13 @@ class TestSmartMatrix(MpfTestCase):
             call(b'\xba\x11\x00\x03\x04\x00\x00\x00\x00\x01\x02\x03')   # frame
             ])
 
+        #test old cookie
         self.machine.rgb_dmds.smartmatrix_2.update([0x00, 0x01, 0x02, 0x03])
         self.advance_time_and_run(.1)
         start = time.time()
         while self.serial_mocks["com5"].write.call_count < 2 and time.time() < start + 10:
             time.sleep(.001)
         self.serial_mocks["com5"].write.assert_has_calls([
-            call(b'\xba\x11\x00\x03\x14\x7f\x00\x00'),                  # brightness
             call(b'\x01\x00\x01\x02\x03')                               # frame
             ])
 

